ci-operator/step-registry/openshift/e2e/test/qe/openshift-e2e-test-qe-chain.yaml

```diff
@@ -2,8 +2,8 @@ chain:
   as: openshift-e2e-test-qe
   steps:
   - ref: cucushift-helper-service-build-info
```

Comment:
We can see there is already a "cucushift-" step before `openshift-extended-test`, so the step `cucushift-pre` is fine to be before too.

Reply:
And there are many "cucushift-" steps referenced by `openshift-tests-private-` stuff. So this time, I want to be as simple as possible to just adjust the order of cucushift-pre. We can enhance if needed once https://issues.redhat.com/browse/OCPQE-10622 using the IDP users in go-lang cases is finally possible in future @jianzhangbjz.

Reply:
ok, yes, splitting the `user setting` from this `cucushift-pre` step will follow `Prow` rules, please don't forget do update this in the future.

Reply:
Sure. Could you share some link about "Prow rules" for me to more clear refer to?
Another question, `cucushift-helper-service-build-info` is a `cucushift-` step before `openshift-extended-test` step, does this follow or not follow "Prow rules"?
A third question, when I run "grep **cucushift** ci-operator/config/openshift/**openshift-tests-private**/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml", the result has many "cucushift" occurrences, "cucushift" stuff is referenced by "openshift-tests-private" stuff, does this follow or not follow "Prow rules"?

Reply:
> Could you share some link about "Prow rules" for me to more clear refer to?

For the step, the simple the better. Generally, the user can know what this step does via reading the step name.

Reply:
> Another question, cucushift-helper-service-build-info is a cucushift- step before openshift-extended-test step, does this follow or not follow "Prow rules"?

Nothing with the Prow rule, but with the agreement with the team automation gogal: replace cucushift with Ginkgo.

> A third question, when I run "grep cucushift ci-operator/config/openshift/openshift-tests-private/openshift-openshift-tests-private-release-4.12__amd64-nightly.yaml", the result has many "cucushift" occurrences, "cucushift" stuff is referenced by "openshift-tests-private" stuff, does this follow or not follow "Prow rules"?

Yes, it's a test run file that contains `cucshift` and `ginkgo` tests. The Prow rule I mean before is about the `step wring`, you can refer to my PR: https://github.com/openshift/release/pull/32626

Reply:
Thanks for clarifying `for the step, the simple the better` for "Prow rules". Got.